docs/install/kubernetes.md

```diff
@@ -154,6 +155,19 @@ When the installation is complete, the helm command prints the URLs and ports of
 - An issue with Prometheus node selector. The workaround for now is to opt out of kube-prometheus-stack by installing the chart with the `--set kube-prometheus-stack.enabled=false`.
 ```
 
+## Configuring TDengine and Kafka for model monitoring
+TDengine and Kafka are part of the default CE installations. These are the default TDengine and Kafka installation values. It's recommended to change the user/password.
+
+```py
+stream_path = "kafka://kafka-stream:9092"
+tsdb_connection = "taosws://root:taosdata@tdengine-tsdb:6041"
+project.set_model_monitoring_credentials(
+    tsdb_connection=tsdb_connection, stream_path=stream_path
+)
+```
```

Comment:
This is not a valid way to set MM credentials. In fact, there are no `tsdb_connection` or `stream_path` parameters in this method:
https://docs.mlrun.org/en/latest/api/mlrun.projects/index.html#mlrun.projects.MlrunProject.set_model_monitoring_credentials

Please fix it :)